packages/frontend/src/components/MkUserInfo.vue

```diff
@@ -29,7 +29,7 @@ SPDX-License-Identifier: AGPL-3.0-only
 			<p :class="$style.statusItemLabel">{{ i18n.ts.followers }}</p><span :class="$style.statusItemValue">{{ number(user.followersCount) }}</span>
 		</div>
 	</div>
-	<MkFollowButton v-if="$i && user.id != $i.id" :class="$style.follow" :user="user" mini/>
+	<MkFollowButton :class="$style.follow" :user="user" mini/>
```

Comment:
Wouldn't it make sense to keep `v-if="user.id != $i.id"` as otherwise the follow button would show if you are logged in and viewing your own info.